metrics/types/types.proto

```diff
@@ -39,3 +39,11 @@ message ViewTimeouts {
   // Number of view timeouts.
   uint64 Timeouts = 3;
 }
+
+message CPUMemoryStats {
+  Event Event = 1;
+  double CPUsagePercentage = 2;
```

Comment:
Either change CPUsagePercentage to CPU**U**sagePercentage or drop "Usage" from both.
I.E. `CPUPercentage` and `MemoryPercentage`